src/detect-pangram.js

```diff
@@ -15,7 +15,21 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const letters = 'abcdefghijklmnopqrstuvwxyz';
+  const toLowerCasePhrase = phrase.toLowerCase();
```

Comment:
если переменная не функция - не может называться действием
```suggestion
  const lowerCasePhrase = phrase.toLowerCase();
```